extensions/amp-video-docking/amp-video-docking.md

```diff
@@ -58,6 +58,8 @@ component's visual area. If the user scrolls back, the video reverts to its orig
 - The video can be flicked to be dismissed from its docked position.
 - Multiple videos on the same page can be docked, but only one at a time will be docked and fixed.
 
+Important: This feature is not supported on `iOS` when served from the [AMP Viewer](https://developers.google.com/search/docs/guides/about-amp#about-google-amp-viewer) due to [Safari restrictions](https://medium.com/@dvoytenko/amp-ios-scrolling-and-position-fixed-b854a5a0d451).
```

Comment:
@dvoytenko – Do you want to link to your 2016 blog post here?

Reply:
I'm confirming if it's still the case. TBH, I was about to publish a post that none of these are hacks are needed anymore :)

Reply:
lol, I'm happy to close this PR and the issue if this is the case - keep me posted :)